Cork/Views/Settings/Panes/Maintenance Pane.swift

```diff
@@ -23,7 +23,7 @@ struct MaintenancePane: View
         {
             VStack(alignment: .leading, spacing: 10)
             {
-                Text("Set default maintenance steps")
+                Text("Default steps")
```

Comment:
The title of this tab is already 'Maintenance', which provides context

Reply:
I'm torn on this one. On one hand, "Default steps" is very succinct to the point. On the other hand, I really wanted to drive home that you're setting the default steps here (because the UI is identical to the one used for the actual mantenance), so I didn't want some users to get confused.

What would you think about "Set default steps"?

Reply:
Ah okay, sure thing! I thought about changing 'steps' to 'tasks' too perhaps, both here and in the maintenance view itself?

Reply:
I think steps would be more approachable for the average user. When I see "task", I imagine something that has a distinct and separate start and end, while "step" is just a part of an overall process

Reply:
Cool!